meson_options.txt

```diff
@@ -25,6 +25,7 @@ option('tvm-support', type: 'feature', value: 'auto')
 option('trix-engine-support', type: 'feature', value: 'auto')
 option('nnstreamer-edge-support', type: 'feature', value: 'auto')
 option('mxnet-support', type: 'feature', value: 'auto')
+option('aitt-support', type: 'feature', value: 'auto')
```

Comment:
Add a comment or option-description to LINE28 that this is a workaround to be removed soon and users shouldn't depend on this option.